### PR TITLE
initial commit, doesnt work yet: issues with incidence proportion (re…

### PR DESCRIPTION
**Pull request by kuenzelt**
Issues with incidence proportions, always returns 1,0.

### CODE REVIEW

File: vignettes/savvyr_example.Rmd [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+```
+
+
+```{r, include=TRUE, echo=TRUE}
+# packages
+packs <- c("data.table", "etm", "survival", "mvna", "knitr", "kableExtra")
+for (i in 1:length(packs)) {
+  library(packs[i], character.only = TRUE)
+}
+```
```
Comment (danielinteractive) [inline vignettes/savvyr_example.Rmd:29]: we should not need this, I would only directly load packages as needed below (only `savvyr` load on top here)

Comment (kuenzelt) [reply vignettes/savvyr_example.Rmd]: adapted

File: vignettes/savvyr_example.Rmd [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+
+
+# Example using dummy data
+
+We generate the dataset $S1$ in  @stegherr_meta_analytic_2021 using the parameter values for Arm A:
+
+```{r, include=TRUE, echo=TRUE}
+# sample size
+N <- 200
```
Comment (danielinteractive) [inline vignettes/savvyr_example.Rmd:34]: maybe here try to comply with linter, so e.g. here
```suggestion
n <- 200
```

File: vignettes/savvyr_example.Rmd [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+# support of uniform censoring distribution
+min.cens <- 0
```
Comment (danielinteractive) [inline vignettes/savvyr_example.Rmd:37]:
```suggestion
min_cens <- 0
```
etc.
Note that the "Rename in Scope" menu entry in RStudio is very helpful for this renaming of variables.

File: vignettes/savvyr_example.Rmd [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+tab <- rbind(IP, ID, KM, IDCE2, AJ2[1:2], IDCE3, AJ3[1:2])
+colnames(tab) <- c("estimated AE probability", "variance of estimation")
+rownames(tab) <- c(
+  "incidence proportion", "probability transform incidence density ignoring competing event",
+  "1 - Kaplan-Meier", "probability transform incidence density (death only)",
+  "Aalen-Johansen (death only), AE risk", "probability transform incidence density (all CEs)",
+  "Aalen-Johansen (all CEs), AE risk"
+)
+
+# probability of AE
```
Comment (danielinteractive) [inline vignettes/savvyr_example.Rmd:88]: generally I would try to move as many (if not all) comments from code into actual text with sentences. This helps the reader a lot to understand the whole workflow better.

Comment (kuenzelt) [reply vignettes/savvyr_example.Rmd]: adapted

File: vignettes/savvyr_example.Rmd (hunk 2 above)
Comment (kuenzelt) [reply vignettes/savvyr_example.Rmd]: adapted

**Review by danielinteractive — APPROVED** [review]
Looks very nice, thanks @kuenzelt !

**Pull request merged**